app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/filtersControl/filtersItem/filtersItem.jsx

```diff
@@ -18,6 +18,7 @@ export class FiltersItem extends PureComponent {
     filter: PropTypes.object.isRequired,
     onChange: PropTypes.func.isRequired,
     onEdit: PropTypes.func.isRequired,
+    editable: PropTypes.bool,
```

Comment:
I think this prop is unnecessary now.
`editable` state is controlled by the component itself (see https://github.com/reportportal/service-ui/pull/1125/files#diff-232b5b52b9b43ec5f2915609e54ad5dbR54)

Just setting `editable={false}` on line 51 (https://github.com/reportportal/service-ui/pull/1125/files#diff-232b5b52b9b43ec5f2915609e54ad5dbR51) won't be a mistake.